app/src/main/java/kth/jjve/xfran/CalendarViewActivity.java

```diff
@@ -66,8 +80,10 @@ protected void onCreate(Bundle savedInstanceState) {
         });
 
         buttonNewEvent.setOnClickListener(v -> {
-            //TODO make it a popup window, not a new activity
-            startActivity(new Intent(this, EditEventActivity.class));
+            Intent intent = new Intent(this, EditEventActivity.class);
```

Comment:
The name EditEventActivity is a bit confusing to me. It is the place where we plan the event right?

in that case, could we refactor it to planEventActivity or something like that?

Reply:
It's where you give it the start and end time (and then save it), that's why editEvent, but I can change it, doesn't really matter